server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariable.java

```diff
@@ -84,6 +84,28 @@ public class TestSqlHostVariable {
           + "       01 {$*VAR-INVALID-BIN`->VAR-INVALID-BIN`->VAR-INVALID-BIN-LEN`->VAR-INVALID-BIN-TEXT} USAGE IS SQL TYPE IS VARBINARY({-|1}1234).\n"
           + "        PROCEDURE DIVISION.\n";
 
+  public static final String LOD_VARS_TEXT =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*VAR-NAME1} USAGE IS SQL TYPE IS BINARY LARGE OBJECT (10).\n"
+                  + "       01 {$*VAR-NAME2} USAGE IS SQL TYPE IS BLOB (10 K).\n"
+                  + "       01 {$*VAR-NAME3} USAGE IS SQL TYPE IS CHARACTER LARGE OBJECT (10 M).\n"
+                  + "       01 {$*VAR-NAME4} USAGE IS SQL TYPE IS CHAR LARGE OBJECT (10 G).\n"
+                  + "       01 {$*VAR-NAME5} USAGE IS SQL TYPE IS CLOB (20).\n"
+                  + "       01 {$*VAR-NAME6} USAGE IS SQL TYPE IS DBCLOB (30 K).\n"
+                  + "       01 {$*VAR-NAME7} USAGE IS SQL TYPE IS BLOB-LOCATOR.\n"
+                  + "       01 {$*VAR-NAME8} USAGE IS SQL TYPE IS CLOB-LOCATOR.\n"
+                  + "       01 {$*VAR-NAME9} USAGE IS SQL TYPE IS DBCLOB-LOCATOR.\n"
+                  + "       01 {$*VAR-NAME10} USAGE IS SQL TYPE IS BLOB-FILE.\n"
+                  + "       01 {$*VAR-NAME11} USAGE IS SQL TYPE IS CLOB-FILE.\n"
+                  + "       01 {$*VAR-NAME12} USAGE IS SQL TYPE IS DBCLOB-FILE.\n"
+                  + "       01 {$*VAR-NAME12} USAGE IS SQL TYPE IS BINARY LARGE OBJECT (10).\n"
```

Comment:
@shciubrcom introduced a mechanism to check the definition location of these implicit variables.
Please check https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariableVarbinary.java